actions/connect.go

```diff
@@ -122,6 +122,10 @@ func (c *Connector) communicate(ctx context.Context, stream actionStreamer, rese
 				break
 			case msg.Sequence == api.Sequence_PING:
 				glog.V(4).Info("Received keep-alive ping from server")
+				err = stream.Send(&api.AgentReply{Sequence: api.Sequence_PING})
```

Comment:
Here it should beok since backend does not respond to PING, else it would be better to have PING and reply to it using PONG (in order to prevent infinite loops).